lib/puppet/provider/dependency.rb

```diff
@@ -1,5 +1,5 @@
 require 'puppet/provider'
-require 'puppet/provider/null_provider'
+require './null_provider'
```

Comment:
I guess these don't get cleared when new code is pushed.

require_relative '../foo'
is basically
require File.join(File.expand_path(File.dirname(__FILE__)), '../foo')